File: src/controle-fluxo-empresarial/src/models/CondicaoPagamento/ParcelaPagamento.ts
import { FormaPagamento } from "./FormaPagamento";

export interface ParcelaPagamento {
    numeroParcela: number;
    formaPagamentoId: number;
    formaPagamento: FormaPagamento;
    dataVencimento: Date

    valor: number;
    desconto: number;
    valorTotal: number;
}

File: src/ControleFluxoEmpresarial/Services/CondicoesPagamento/models/ParcelaPagamento.cs
using ControleFluxoEmpresarial.Models.CondicaoPagamentos;
using System;

namespace ControleFluxoEmpresarial.Services.CondicoesPagamento.models
{
    public class ParcelaPagamento
    {
        public int NumeroParcela { get; set; }

        public decimal Valor { get; set; }
        public decimal Desconto { get; set; }
        public decimal ValorTotal { get; set; }

        public int FormaPagamentoId { get; set; }
        public FormaPagamento FormaPagamento { get; set; }

        public DateTime DataVencimento { get; set; }
    }
}
